client/extensions/woocommerce/state/sites/settings/email/selectors.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import { get, isArray } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+/**
+ * @param {Object} state Whole Redux state tree
+ * @param {Number} [siteId] Site ID to check. If not provided, the Site ID selected in the UI will be used
+ * @return {Array} Email settings.
+ */
+export const getEmailSettings = ( state, siteId = getSelectedSiteId( state ) ) => {
+	return get( state, [ 'extensions', 'woocommerce', 'sites', siteId, 'settings', 'email' ] );
```

Comment:
Should this return null or something if this part of the state tree is not found or loaded yet? Looks like null is what you set at https://github.com/Automattic/wp-calypso/pull/19802/files#diff-6d9c2cfa547c7f0cf2499d904a760ecbR102